docker-compose.yml

```diff
@@ -8,12 +8,8 @@ services:
       args:
         NODE_ENV: development
     command: yarn run start-watch
-    volumes:
-      - .:/code
```

Comment:
Why has this been removed? I think the result of not having this here is changes made to the code are not going to be picked up in the `test-watch` and `start-watch` scripts when run in the Docker container.

Reply:
mounting the source code folder replaces the code folder in the docker image, which breaks the CI tests build as there is no header-items.nunjucks in the source folder.

I'll add the volume specifically to the start and test sections to ensure changes are seen. These include the header-build as part of the task, to ensure the file is present.

Reply:
I can see the point of removing the mapping in `docker-compose-tests` but in this file I think it is required. Or this needs moving from the `base-app` service to the `frontend` service.